recipes/_lustre_install.rb

```diff
@@ -47,4 +47,11 @@
   end
 elsif node['platform'] == 'centos'
   Chef::Log.warn("Unsupported version of Centos, #{node['platform_version']}, supported versions are 7.6 and 7.5")
-end
+elsif node['platform'] == 'amazon'
+
+  # Install lustre client module
+  yum_package 'lustre-client' do
```

Comment:
we should use "package" instead of the declined version for the OS, see https://docs.chef.io/resource_package.html and https://docs.chef.io/resource_yum_package.html
```
package lustre-client do
    retries 3
    retry_delay 5
  end
```

Reply:
Good Idea 👍🏻

Reply:
added